static/bidder-info/greedygame.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "http://ads-pbs.rtb-greedygame.com/openrtb/{{.PublisherID}}"
+maintainer:
+  email: "engineering@project-limelight.com"
```

Comment:
Just a note for other reviewers - This email address has been validated on Aug 28th so skipping to send a verification email this time.